src/Lockedfield.php

```diff
@@ -178,16 +178,7 @@ public function getLocks($itemtype, $items_id, bool $fields_only = true)
             if ($fields_only === true) {
                 $locks[] = $row['field'];
             } else {
-                $value = $row['value'];
-                //if it refers to a ForeignKey get dropdown name
-                if (
-                    $row['value'] !== null
-                    && isForeignKeyField($row['field'])
-                    && is_a($itemtype = getItemtypeForForeignKeyField($row['field']), CommonDropdown::class, true)
-                ) {
-                    $value = Dropdown::getDropdownName(getTableForItemType($itemtype), $row['value'], 0, true, false, $row['value']);
-                }
```

Comment:
This logic should be put back somewhere (in the twig template maybe), in order to hace dropdown names displayed in lock icon tooltip.

Reply:
In my tests, all values were displayed corrdctly. We do not want to display dropdowns names here, but inventory value.

Reply:
I just add such a case in test suite.